src/query/sql/src/planner/binder/copy_into_location.rs

```diff
@@ -59,8 +59,18 @@ impl<'a> Binder {
                 }
             }
             CopyIntoLocationSource::Query(query) => {
-                self.bind_statement(bind_context, &Statement::Query(query.clone()))
-                    .await
+                let select_plan = self
+                    .bind_statement(bind_context, &Statement::Query(query.clone()))
+                    .await?;
+                if let Plan::Query { s_expr, .. } = &select_plan {
+                    if !self.check_sexpr_top(s_expr, super::binder::CheckType::Copy)? {
```

Comment:
copyIntoTable has limititions . not CopyIntoLocation
https://docs.snowflake.com/en/user-guide/data-load-transform#supported-functions

Reply:
yes, but here we need to limit project set and udf, I'm refactor. please wait. cc @youngsofun